### PR TITLE
Fixes #13602: Add a technique to manage basic SNMP configuration

### PR DESCRIPTION
**Pull request by Fdall**
https://www.rudder-project.org/redmine/issues/13602

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

File: techniques/applications/snmp/1.0/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+
+<TECHNIQUE name="SNMP installation">
+  <DESCRIPTION>This technique installs an SNMP agent, and configures it
```
Comment (amousset) [inline techniques/applications/snmp/1.0/metadata.xml]: s/This technique//

**Comment by Fdall** [comment]
Commit modified

File: techniques/applications/snmp/1.0/snmp.st [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        "service_reload_prefix" string => "service_reload_${canonified_snmp_service}";
+        "service_running_prefix" string => "service_ensure_running_${canonified_snmp_service}";
+        "block_prefix" string => "file_ensure_block_in_section_${canonified_snmp_conf_file}";
+
+
+  classes:
+      "pass3" expression => "pass2";
+      "pass2" expression => "pass1";
+      "pass1" expression => "any";
+      "public_community" expression => strcmp("${acl[${community_indice}]}", "rocommunity");
```
[the remark this answers is not in the document]
Comment (Fdall) [reply techniques/applications/snmp/1.0/snmp.st]: Nevermind, those lines are unused...

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
OK, merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Pull request merged**